scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableSyntaxConfig.scala

```diff
@@ -32,8 +32,10 @@ case class DisableSyntaxConfig(
     noValInAbstract: Boolean = false,
     @Description("Report error on object with `implicit` modifier.")
     noImplicitObject: Boolean = false,
-    @Description("Report error on method that define an implicit conversion. ")
+    @Description("Report error on method that define an implicit conversion.")
     noImplicitConversion: Boolean = false,
+    @Description("Report error on usage of the constructor Some.")
+    noSome: Boolean = false,
```

Comment:
I think this is spurious. You're adding a config option to the `DisableSyntax` rule, but this then goes unused by the rule itself.